factory/api/apiResolverFactory.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/node/external/transactionAPI"
 	"github.com/ElrondNetwork/elrond-go/node/trieIterators"
 	trieIteratorsFactory "github.com/ElrondNetwork/elrond-go/node/trieIterators/factory"
+	"github.com/ElrondNetwork/elrond-go/outport/process/alteredaccounts"
```

Comment:
I think for some other packages, we use the `alteredAccounts` style. But can be either way.

Reply:
the standard is `alteredaccounts`